NOTICES.txt

```diff
@@ -157,7 +157,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 
 
->>> github.com/cyberark/conjur-authn-k8s-client-0.13.0
+>>> github.com/cyberark/conjur-authn-k8s-client-0.15.0
 
 Copyright [yyyy] [name of copyright owner]
```

Comment:
the copyright actually changed in this version, can you please copy/paste the updated one here?

Reply:
done. nice catch.